### PR TITLE
Enable the upload of ETW traces to CLR CAP in Windows daily build

### PR DESCRIPTION
**Pull request by daxian-dbw**
# PR Summary

A variable group was created that contains the variable `CapIngressToken`, which stores the ingress token used for the upload.
Update the `windows-daily.yml` file to provide the ingress token when calling `Stop-TraceCollection`, so the upload can happen.

Here is the build that validated the changes in this PR:
https://dev.azure.com/powershell/PowerShell/_build/results?buildId=54570&view=results

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

File: .vsts-ci/windows-daily.yml [hunk 1]
```diff
@@ -48,6 +48,8 @@ stages:
```
Comment (PoshChan) [inline .vsts-ci/windows-daily.yml:48]: @daxian-dbw, your last commit had 1 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://powershell.visualstudio.com/PowerShell/_build/latest?definitionId=32 should work
```powershell
Failed to complete request to "https://powershell.visualstudio.com/PowerShell/_build/latest?definitionId=32". Response status code does not indicate success: 404 (Not Found).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 119
119:                                     throw "Failed to complete request to `"$url`". $($_.Exception.Message)"

```

Comment (daxian-dbw) [reply .vsts-ci/windows-daily.yml:48]: This will be fixed by #12887

**Comment by daxian-dbw** [comment]
Make this PR a draft because there seems to be an issue in uploading the trace to //CLRCAP.

File: .vsts-ci/windows-daily.yml (hunk 1 above)
Comment (PoshChan) [inline .vsts-ci/windows-daily.yml:48]: @daxian-dbw, your last commit had 1 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://powershell.visualstudio.com/PowerShell/_build/latest?definitionId=32 should work
```powershell
Failed to complete request to "https://powershell.visualstudio.com/PowerShell/_build/latest?definitionId=32". Response status code does not indicate success: 404 (Not Found).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 119
119:                                     throw "Failed to complete request to `"$url`". $($_.Exception.Message)"

```

**Comment by daxian-dbw** [comment]
@PoshChan Please retry static

**Comment by PoshChan** [comment]
@daxian-dbw, successfully started retry of `PowerShell-CI-static-analysis`

**Comment by daxian-dbw** [comment]
Contacted the CAP team and it turns out the trace files were uploaded corrected from my test runs of Windows-daily build (see below telemetry from the CAP side).
So the changes to `windows-daily.yml` is good to go.

image

**Review by adityapatwardhan — APPROVED** [review]

**Pull request merged**